src/components/elements/input/LabeledCheckbox.tsx

```diff
@@ -71,7 +73,8 @@ const LabeledCheckbox = ({
             ...labelSx,
           }}
           onChange={onChange}
-          {...props}
+          aria-label={ariaLabel}
```

Comment:
We only really need aria label on the checkbox for some reason the checkbox label is hidden. And in that case it should be passed to the `Checkbox` so it ends up on the `input` component. I don't think there is any point in adding an `aria-label` to a label like this.